resources/assets/js/components/FriendsList.vue

```diff
@@ -129,13 +129,16 @@
                     text: 'This will delete @' + friend.username + ' from your list of friends',
                     type: 'warning',
                     showCancelButton: true
-                }, function () {
+                }, () => {
                     vm.$http.delete('/api/conferences/' + vm.conferenceId + '/' + vm.key + '/' + friend.id)
-                        .then(function () {
-                            vm.list.$remove(friend)
-                        });
+                        .then(() => { vm.list.$remove(friend) });
                 });
             },
+
+            metFriend: function (friend) {
+                this.$http.patch('/api/conferences/' + this.conferenceId + '/' + this.key + '/' + friend.id, { met: ! friend.met })
```

Comment:
Gotcha! So, two options:

1. Create `markFriendMet` and `markFriendNotMet`, separating the calls based on whether or not `friend.met` evaluates to `true`
2. Change `metFriend` to `toggleFriend` (which just wraps the above methods)

Which do you prefer (or have opinions on other approaches)?